libcextract/Passes.cpp

```diff
@@ -438,12 +438,11 @@ class FunctionExternalizerPass : public Pass
     {
       /* Issue externalization.  */
       SymbolExternalizer externalizer(ctx->AST.get(), ctx->IA, ctx->Ibt, ctx->PatchObject, ctx->DumpPasses);
-      externalizer.Externalize_Symbols(ctx->Externalize);
-      if (ctx->RenameSymbols) {
-        /* The FuncExtractNames will be modified, as the function will be
-           renamed.  */
-        externalizer.Rename_Symbols(ctx->FuncExtractNames);
-      }
+      if (ctx->RenameSymbols)
+        externalizer.Externalize_Symbols(ctx->Externalize, ctx->FuncExtractNames);
```

Comment:
Please keep the comment regarding the fact that `ctx->FuncExtractNames` is modified in `Externalize_Symbols`. This behavior is not trivial.